public/view_lecture_teams.php

```diff
@@ -50,22 +50,20 @@
            <table>
                 <thead>
                     <tr>
-                        <th>#</th>
+                        <th>Student ID</th>
                         <th>First Name</th>
                         <th>Last Name</th>
                         <th>Email Address</th>
-                        <th>Created At</th>
                     </tr>
                 </thead>
                 <tbody>
                 <?php $students_in_team = $row->team_members;
                 foreach ($students_in_team as $student) {?>
                 <tr>
-                    <td><?php echo escape($student->user_id); ?></td>
+                    <td><?php echo escape($student->user->student_id); ?></td>
```

Comment:
can you help add the `includes` for users as well 
<img width="1311" alt="image" src="https://user-images.githubusercontent.com/44499264/164529771-8e38b188-52c9-454a-be68-8f618cc92226.png">

Reply:
done